functions/New-DbaAvailabilityGroup.ps1

```diff
@@ -252,6 +265,7 @@ function New-DbaAvailabilityGroup {
         [ValidateSet('Automatic', 'Manual')]
         [string]$SeedingMode = 'Manual',
         [string]$Endpoint,
+        [string[]]$EndpointUrl,
```

Comment:
See comments on Add function as this command should have the same rules applied. It is there to create the AG, so we simply need to ensure the endpoint name they provide already exists and if it does not point them to New-DbaEndpoint. That allows them to use FQDN, IP, or whatever else they want.

Reply:
Please dig a little deeper into this topic and then we will continue this discussion.